src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsFandomObject.java

```diff
@@ -7,116 +7,129 @@
-  }
-
-  public void verifyFandomMobileArticleBottomLeaderboard() {
+  public void verifyFandomBottomLeaderboardDesktop() {
     jsActions.scrollToElement(wait.forElementVisible(By.cssSelector(
-        presentFandomMobileArticleBottomLeaderboardSelector)));
-    verifyAdVisibleInSlot(presentFandomMobileArticleBottomLeaderboardSelector,
-                          presentFandomMobileArticleBottomLeaderboard);
+        presentFandomBottomLeaderboardDesktopSelector)));
+    verifyAdVisibleInSlot(presentFandomBottomLeaderboardDesktopSelector,
+                          presentFandomBottomLeaderboardDesktop);
   }
 
-  public void verifyFandomHubTopLeaderboard() {
-    verifyAdVisibleInSlot(presentFandomHubTopLeaderboardSelector,
-                          presentFandomHubTopLeaderboard);
+  public void verifyFandomBottomLeaderboardMobile() {
+    jsActions.scrollToElement(wait.forElementVisible(By.cssSelector(
+        presentFandomBottomLeaderboardMobileSelector)));
+    verifyAdVisibleInSlot(presentFandomBottomLeaderboardMobileSelector,
+                          presentFandomBottomLeaderboardMobile);
   }
 
-  public void verifyFandomHubBottomLeaderboard() {
+  public void verifyFandomTopBoxad() {
     jsActions.scrollToElement(wait.forElementVisible(By.cssSelector(
-        presentFandomHubBottomLeaderboardSelector)));
-    verifyAdVisibleInSlot(presentFandomHubBottomLeaderboardSelector,
-                          presentFandomHubBottomLeaderboard);
+        presentFandomTopBoxadSelector)));
+    verifyAdVisibleInSlot(presentFandomTopBoxadSelector, presentFandomTopBoxad);
   }
 
-  public void verifyFandomDesktopTopBoxad() {
+  public void verifyFandomTopBoxadDesktop() {
     jsActions.scrollToElement(wait.forElementVisible(By.cssSelector(
         presentFandomTopBoxadDesktopSelector)));
     verifyAdVisibleInSlot(presentFandomTopBoxadDesktopSelector, presentFandomTopBoxadDesktop);
   }
 
-  public void verifyFandomMobileTopBoxad() {
+  public void verifyFandomTopBoxadMobile() {
     jsActions.scrollToElement(wait.forElementVisible(By.cssSelector(
         presentFandomTopBoxadMobileSelector)));
     verifyAdVisibleInSlot(presentFandomTopBoxadMobileSelector, presentFandomTopBoxadMobile);
   }
 
-  public void verifyFandomTopBoxad() {
-    jsActions.scrollToElement(wait.forElementVisible(By.cssSelector(
-        presentFandomTopBoxadSelector)));
-    verifyAdVisibleInSlot(presentFandomTopBoxadSelector, presentFandomTopBoxad);
-  }
-
   public void verifyFandomBottomBoxad() {
     jsActions.scrollToElement(wait.forElementVisible(By.cssSelector(
         presentFandomBottomBoxadSelector)));
     verifyAdVisibleInSlot(presentFandomBottomBoxadSelector, presentFandomBottomBoxad);
   }
 
+  public void verifyNoFandomBottomBoxadMobile() {
+    jsActions.scrollToElement(wait.forElementVisible(By.cssSelector(
+        presentFandomBottomBoxadMobileSelector)));
+    verifyNoAd(presentFandomBottomBoxadMobileSelector + " > div");
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal " > div" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1192)